sqlstore/src/main/resources/db/migration/V5201__PacBio_library_options.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO LibraryType (description, platformType, archived) VALUES ('Whole Genome', 'PACBIO', 0);
+
+INSERT INTO LibrarySelectionType (name, description) VALUES ('SageHLS', 'Sage High Molecular Weight Library System'), ('BluePippin', 'Sage BluePippin size selection'), ('None', 'None');
```

Comment:
'Sage BluePippin' is indeed distinct, and Jeremy was clear that 'size fractionation' was not appropriate for PacBio.

Instead of adding 'None', I'll update the UI to be able to select a default null '(None)'.